### PR TITLE
Connecting frontend and backend to receive simple translations 

### PR DESCRIPTION
**Pull request by cjoshea9**
Allows data to be displayed from the backend in the output textbox.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,35 +2,73 @@ import React from 'react';
...
+    fetch('https://cjsback.herokuapp.com/', {
+      method: 'POST',
+      headers: new Headers({
+        'Content-Type': 'application/x-www-form-urlencoded',
+      }),
+      body: "input="+ this.state.input
+    })
+      .then(res => res.json())
+      .then((data) => {
+        console.log(data)
```
Comment (Danish21) [inline src/App.js]: Remove?

File: src/App.js [hunk 2]
```diff
@@ -2,35 +2,73 @@ import React from 'react';
...
+          output: data['response']
+        })
+      })
+  }
+
+  updateInput(input) {
+    this.setState({
+      input: input.target.value
+    })
+    console.log(this.state.input)
```
Comment (Danish21) [inline src/App.js]: Remove?

**Review by Danish21 — APPROVED** [review]
Nice!

File: src/App.js (hunk 2 above)
Comment (cjoshea9) [reply src/App.js]: Yes I can remove those.

**Comment by cjoshea9** [comment]
Screenshot 2020-06-24 at 3 15 49 PM

**Pull request merged**